rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- [ ] You have only added date as the initialize parameter whereas you have book and person too, kindly add them too.